web/admin/application/configs/klear/model/Users.yaml

```diff
@@ -403,6 +403,29 @@ production:
         icon: help
         text: _("Add QR Code to user portal to provision GS Wave mobile softphone")
         label: _("Need help?")
+    multiContact:
+      title: _('Multi contact')
+      type: select
+      defautlValue: 1
+      source:
+        data: inline
+        values:
+          '0':
+            title: _("No")
+            visualFilter:
+              show: [""]
```

Comment:
FTR: Quotes are not required in empty lists, `[]` is perfectly fine.
Anyway, no need to change that in this PR.